examples/server/server.cpp

```diff
@@ -430,6 +441,39 @@ int main(int argc, const char ** argv) {
     svr.reset(new httplib::Server());
 #endif
 
+    std::unordered_map<std::string, std::string> model_map = {};
+    const std::string model_path = args.get_string_param("--model-path");
+    if (std::filesystem::is_directory(model_path)) {
+        for (auto const &entry : std::filesystem::directory_iterator(model_path)) {
+            if (!entry.is_directory()) {
```

Comment:
```suggestion
            if (!entry.is_directory() && entry.path().extension() == ".gguf") {
```

Don't crash if there's a `README.md`

Reply:
ty!